ci_framework/playbooks/05-build-operators.yml

```diff
@@ -17,6 +17,7 @@
         oc registry login --insecure=true
 
     - name: Build operator and meta-operator
+      when: cifmw_operator_build_operators is defined
```

Comment:
MAYBE a double-check:
- is defined
- | length > 0
Just to be on the safe side..